exercises/pangram/canonical-data.json

```diff
@@ -3,66 +3,78 @@
           "property": "isPangram",
           "input": "",
           "expected": false
         },
         {
-          "description": "pangram with only lower case",
+          "description": "returns false for an undefined/null message as argument.",
           "property": "isPangram",
-          "input": "the quick brown fox jumps over the lazy dog",
+          "input": null,
+          "expected": false
+        },
+        {
+          "description": "recognizes a perfect lower case pangram.",
+          "property": "isPangram",
+          "input": "abcdefghijklmnopqrstuvwxyz",
           "expected": true
         },
         {
-          "description": "missing character 'x'",
+          "description": "recognizes missing characters, e.g. 'x'.",
           "property": "isPangram",
-          "input": "a quick movement of the enemy will jeopardize five gunboats",
+          "input": "aquickmovementoftheenemywilljeopardizefivegunboats",
           "expected": false
         },
         {
-          "description": "another missing character 'x'",
+          "description": "recognizes missing characters, e.g. 'h'.",
           "property": "isPangram",
-          "input": "the quick brown fish jumps over the lazy dog",
+          "input": "fiveboxingwizardsjumpquicklyatit",
           "expected": false
         },
         {
-          "description": "pangram with underscores",
+          "description": "recognizes an upper-case pangram.",
           "property": "isPangram",
-          "input": "the_quick_brown_fox_jumps_over_the_lazy_dog",
+          "input": "THEQUICKBROWNFOXJUMPSOVERTHELAZYDOG",
           "expected": true
         },
         {
-          "description": "pangram with numbers",
+          "description": "recognizes a missing character in upper-case, e.g. 'G'.",
+          "property": "isPangram",
+          "input": "THEQUICKBROWNFOXJUMPSOVERTHELAZYDOLL",
+          "expected": false
+        },
+        {
+          "description": "recognizes a mixed-case pangram.",
           "property": "isPangram",
-          "input": "the 1 quick brown fox jumps over the 2 lazy dogs",
+          "input": "ThEFiVebOxInGWiZaRdSJuMpqUiCkLy",
           "expected": true
         },
         {
-          "description": "missing letters replaced by numbers",
+          "description": "recognizes a missing character in mixed-case, e.g. 'Y'.",
           "property": "isPangram",
-          "input": "7h3 qu1ck brown fox jumps ov3r 7h3 lazy dog",
+          "input": "ThEFiVebOxInGWiZaRdSJuMpqUiCkL",
           "expected": false
         },
         {
-          "description": "pangram with mixed case and punctuation",
+          "description": "recognizes a pangram even if additional characters other than a-z are present.",
           "property": "isPangram",
-          "input": "\"Five quacking Zephyrs jolt my wax bed.\"",
+          "input": "Victor_jagt-zwölf.B0xkämpfer >qu3r<über;den'großen\" Sylter#Deich!",
```

Comment:
can you confirm whether or not ö ä ü and ß are in ASCII, given the declaration of intent in https://github.com/exercism/problem-specifications/pull/893#discussion_r139159361 of

> OK, I'll remove non-ASCII tests.